### PR TITLE
Extract `language` from `metadata.language`

### PR DESCRIPTION
**Pull request by amoore108**
Overall changes
======
- Extracts the `language` value for an asset from the `metadata.language` property instead of the `metadata.passport.language` property, as the Passport version does not indicate the correct language tag for a transliterated asset

Testing
======
1. _List the steps used to test this PR._

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Review by emilysaffron — APPROVED** [review]

File: src/app/components/ATIAnalytics/index.test.tsx [hunk 1]
```diff
@@ -306,6 +306,7 @@ describe('ATI Analytics Container', () => {
         lng: 'en-US',
         x2: '[responsive]',
         x3: '[news]',
+        x4: '[en-gb]',
```
Comment (karinathomasbbc) [inline src/app/components/ATIAnalytics/index.test.tsx:309]: Is this OK / as expected?

**Review by karinathomasbbc — APPROVED** [review]

File: src/app/components/ATIAnalytics/index.test.tsx (hunk 1 above)
Comment (amoore108) [reply src/app/components/ATIAnalytics/index.test.tsx:309]: Seems ok. This is taken from a production News AMP article:
Screenshot 2024-08-14 at 13 21 01

Comment (amoore108) [reply src/app/components/ATIAnalytics/index.test.tsx:309]: This test has `x2` as `responsive` which is canonical, but we won't render canonical News articles.

Comment (karinathomasbbc) [reply src/app/components/ATIAnalytics/index.test.tsx:309]: Ya, we can render canonical news on local, but not Test / Live 😄

**Pull request merged**